README.md

```diff
@@ -1,2 +1,48 @@
 # Minio-OpenShift
-Minio deployment template for Rahti 
+
+This template deploys a private object store using [Minio](https://min.io/) in [Rahti](https://rahti.csc.fi/), CSC's OpenShift cluster. The template creates a single pod deployment for Minio, which uses PVC as a backend volume for storing data. PVC size is provided to template as parameters & existing PVC could be also used. Please follow [Minio User Guide](https://www.markdownguide.org/basic-syntax/) for usage of Minio object store.
+
+## Usage
```

Comment:
Possibly ditch Usage-chapter altogether?

Reply:
If it doesn't harm lets keep it here for now, we can remove Usage chapter excluding Parameters table when we add it to Rahti user guide

Reply:
@juhanikataja so all the changes except removal of Usage section in Readme have been committed, Pls review PR & merge devel to Master if they are ok.
P.S.: I have done sanity testing of latest mini template & it works fine